Makefile

```diff
@@ -40,11 +40,13 @@ build-all: .goreleaser.yaml $(GORELEASER) $(MAIN)
 	$(GORELEASER) build --snapshot --clean
 generate: $(MAIN) $(CP_FILES_DEST)
 test: $(BIN)
+	@result=0; \
 	for MOD in $(GOMODULES); do \
 		cd $${MOD}; \
-		go test ./...; \
+		go test -v ./... || result=1; \
 		cd -; \
-	done
+	done; \
+	exit $$result;
```

Comment:
this seems somewhat hacky and can probably be improved, but i guess if it works, it ain't stupid 😄

Reply:
it won't continue without it, didn't manage to find a better solution for now :/

Reply:
The idea is that all tests run, even if one of them fails?

Reply:
yes exactly